gems/sorbet-runtime/lib/types/private/methods/call_validation_2_6.rb

```diff
@@ -1,6 +1,14 @@
+# frozen_string_literal: true
+# typed: false
+
+# DO NOT EDIT. This file is autogenerated. To regenerate, run:
+#
+#     bazel test //gems/sorbet-runtime:update_call_validation
+
+module T::Private::Methods::CallValidation
```

Comment:
This is the same namespace as `call_validation.rb` sets up (i.e., it reopens this module).

If you prefer, I can instead generate the header and footer, and make `call_validation.rb` literally nothing more than a conditional require statement.

Reply:
This is fine; I have no particular preference.